.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: PR check
+on:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Clone source code
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Docker Buildx
+      uses: crazy-max/ghaction-docker-buildx@v1.4.0
+      with:
+        version: v0.4.1
```

Comment:
https://github.com/crazy-max/ghaction-docker-buildx#moved-to-docker-organization